tools/marvin/setup.py

```diff
@@ -46,7 +46,7 @@
                 "marvin.sandbox.basic"],
       license="LICENSE.txt",
       install_requires=[
-          "mysql-connector-python >= 1.1.6",
+          "mysql-connector-python == 8.0.20",
```

Comment:
dare we use >= here?

Reply:
quick stackoverflow search: `"mysql-connector-python>=8.0.20,<8.0.30"` should work

Reply:
Yup that may work as well; but we might be specific like we are with the pom.xml. As with EL8, cpython version is 3.6 we want to use a dependency that is certified for cpython3.6 ; see https://pypi.org/project/mysql-connector-python/8.0.20/#files supports both cp3.6 and cp2.7. Happy to include your workaround as well. I think 8.0.30 and 8.0.31 might work, it's the new 8.0.32 that breaks. I see 8.0.31 used here https://github.com/apache/cloudstack/blob/main/packaging/centos8/cloud.spec#L309 (we bundle these for cloudstack-management)

Reply:
the version might be not important. we just need a version which works.

by the way, is there EOL date of the versions ?

Reply:
let's pick the newest that works

Reply:
done